### ISSUE TITLE
Discrete_Annual_Broadscale should indicate the depth

### ISSUE THREAD

**Issue by clayton33** [casaultb/azmpdata#26]
The help for this dataset currently states, for both temperature and salinity :

```
salinity
Discrete salinity measurements over a range of depths

sea_temperature
Discrete sea temperature measurements over a range of depths
```

Therefore, when we load the dataset, it is expected that there is a depth column, but that is not the case ...

``` r
library(azmpdata)
#> 
#>  casaultb/azmpdata status:
#>  (Package ver: 0.2019.0.9100) Up to date
#>  (Data ver:2021-01-14 ) is up to date
data("Discrete_Annual_Broadscale")
str(Discrete_Annual_Broadscale)
#> 'data.frame':    144 obs. of  4 variables:
#>  $ year           : int  1948 1949 1950 1951 1952 1953 1954 1955 1956 1957 ...
#>  $ area           : chr  "scotian_shelf_box" "scotian_shelf_box" "scotian_shelf_box" "scotian_shelf_box" ...
#>  $ sea_temperature: num  -1.396 0.545 -0.373 1.364 0.5 ...
#>  $ salinity       : num  NA NA NA NA NA NA NA NA NA NA ...
```

<sup>Created on 2021-05-13 by the [reprex package](https://reprex.tidyverse.org) (v2.0.0)</sup>

And actually, I'm not even sure that this is the appropriate classification of this data set. The bottom temperture from the same analysis is in Derived_Annual_Broadscale.

@casaultb we should look back to our original documents where we identified which data belongs to which data set to see if this is what we decided. If it is, then I propose that we put it in Derived_Annual_Broadscale. I'm fairly certain that the cf naming conventions has something for surface values, but we'll have to see if this is associated with satellite like data, or just general surface data.

**Comment by casaultb** [comment]
If I'm not mistaken, the values in this dataset are anomalies of the mean surface temperature and salinity averaged over the Scotian Shelf.
A few things that are incorrect with that dataset:
1- Datasets are to include variables in absolute units (e.g. temperature in deg. C or salinity in psu) but not anomalies. Anomalies are to be calculated externally (e.g. in multiVAR) or as per user specific needs (e.g. over a different reference period). This dataset should be reviewed to include the mean surface temperature and salinity values.
2- The data for this dataset should be part of the Derived_Annual_Broadscale dataset since it is a derived value (i.e. averaged over an area). Datasets labelled with the "Discrete" identifier should only contain _in situ_ measurements and therefore they are also limited in scope to "Occupations" level data.
3- The documentation is incorrect - there is no depth associated with those values. The documentation needs to be updated according to item 1 above.

**Comment by clayton33** [comment]
In response to your points :

1. Now that I double checked this, you're correct, and I think we've discussed it before. For this analysis, it involves calculating monthly anomalies for defined areas, and then doing an area-weighted average of the anomalies to get an annual anomaly for the entire scotian shelf. I was only left with a climatology mean for sigmaTheta for this type of analysis, and when I tried to re-calculate it, I could not get it to match, so I left it for when I need to update the climatology as I didn't need the salinity and temperature climatology values in my reporting.
2. Ok, that answers that. I'll depreciate this data set.
3. But there is a depth associated with these values. When this data set is incorporated into `Derived_Annual_Broadscale`, I'll label these variables as `salinity_0` and `sea_temperature_0`. This naming logic follows what we've done with other datasets, *e.g.* temperature at 0m and 90m at Prince 5 in `Derived_Annual_Stations`.

So in summary I will
- [ ] Depreciate `Discrete_Annual_Broadscale`
- [ ] Incorporate the data into `Derived_Annual_Broadscale`
- [ ] Re-name the variables to reflect the depth that they are associated with.

I'll leave this issue open, until I do a pull request to reflect the above.
